backend/doc/cli.md

```diff
@@ -34,6 +34,22 @@ Useful to migrate old databases, to run before applying v3.0.0 migration scripts
 
     tracimcli db update-naming-conventions
 
+### Migrate Mysql/Mariadb database to utf8mb4 (added for tracim 3.7)
+
+:warning: This will modify the character set and collation of Tracim's database, including the already existing tables. We strongly suggest you to do a backup before applying this command.
```

Comment:
Can we purpose command or a link to official documentation to make this backup ?

Reply:
```suggestion
:warning: This will modify the character set and collation of Tracim's database, including the already existing tables. We strongly suggest you to do a backup of the database before applying this command. We suggest you do follow official documentation of your database to do so.
```